Puppetfile

```diff
@@ -3,6 +3,7 @@ forge "http://forge.puppetlabs.com"
 # Modules from the Puppet Forge
 # Note the versions are all set to :latest but after you've 
 # installed you should change them to the exact version you want
+mod "hunner/hiera", :latest
```

Comment:
Can we change this to whatever the current working version is?  

If something causes our code here to stop working in a newer version of hunner/hiera I'd like to be pinned to a known good version.